src/main/java/com/gmail/yuramitryahin/config/AppConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.gmail.yuramitryahin.config;
```

Comment:
com.gmail?

Reply:
yep, I read that you can specify your mail only in the reverse order of words

Reply:
it's default package

Reply:
I did this in all my previous homework)